Xamarin.Forms.Core/Editor.cs

```diff
@@ -1,7 +1,13 @@
 using System;
 using System.ComponentModel;
+using Xamarin.Forms;
 using Xamarin.Forms.Internals;
 using Xamarin.Forms.Platform;
+using Xamarin.Forms.StyleSheets;
+
+[assembly: StyleProperty("-xf-editor-text-color", typeof(Editor), nameof(Editor.TextColorProperty))]
+[assembly: StyleProperty("-xf-editor-placeholder", typeof(Editor), nameof(Editor.PlaceholderProperty))]
+[assembly: StyleProperty("-xf-editor-placeholder-color", typeof(Editor), nameof(Editor.PlaceholderColorProperty))]
```

Comment:
if we do this, I'd use `xf-placeholder-color`, but I'd like to investigate what it'd take to implement `::placeholder` pseudo-element

Reply:
Will drop it. Well, I'll wait for @jassmith to chime in. Last I understood, he had a very liberal stance on what should be exposed as a vendor property. Basically, not an either or, but why not both kinda position.

Reply:
`-xf-placeholder-color`